src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/WebWorker.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Runtime.InteropServices.JavaScript
     /// This is draft for possible public API of browser thread (web worker) dedicated to JS interop workloads.
     /// The method names are unique to make it easy to call them via reflection for now. All of them should be just `RunAsync` probably.
     /// </summary>
-    internal static class WebWorker
+    public static class WebWorker
```

Comment:
needs API review

Reply:
Yes, but not on this PR. Because I don't mirror the same in the ref assemblies, this is invisible to external code. There is CompatibilitySuppressions.xml which supresses the check for that. 
I could as well remove the class from the PR, if this is too offensive.

Reply:
oh, interesting. it's ok to leave it in, I think. thanks